modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/index.js

```diff
@@ -215,6 +227,19 @@ const Renderer = ({
 				page={page}
 				pageIndex={pageIndex}
 			>
+				{hasRequiredIconDescription && (
+					<p aria-hidden="true" className="text-secondary">
+						{Liferay.Language.get(
+							'required-icon-fields-description-first'
+						)}
+						<span className="c-mr-1 reference-mark">
+							<ClayIcon symbol="asterisk" />
+						</span>
+						{Liferay.Language.get(
+							'required-icon-fields-description-second'
+						)}
```

Comment:
@georgel-pop-lr and @marcosapmf, can we use something similar to this [code](https://github.com/liferay/liferay-portal/blob/7e8a0864a6c4e404f71d0da99bbad36cfb93621a/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/handlers/fieldDuplicatedHandler.es.js#L32-L37)?

Reply:
Hi @natocesarrego,

I already discussed this in this https://issues.liferay.com/browse/PTR-2034 not sure if is possible, but we need to be aware that we need to have a String to use sub, and convert ClayIcon to a String and not a react component. 

Regards.

Reply:
@natocesarrego, as @georgel-pop-lr said, in this case we are adding a ClayIcon between the strings. I'm afraid we can't pass it as a parameter to the language key.

Reply:
Hi all,
For 7.2.x do I need to open another PTR if I need help with the backport? 
I ask this because the solution is pretty much different and I have some problems getting this into 7.2.x. 
Regards.

Reply:
Thanks for your clarification, @georgel-pop-lr and @marcosapmf!

Must we use ClayIcon for this specific use case? What are the drawbacks of not using it for this use case?

Reply:
Well I guess will not be so clear what Icon we are referring to, do we have any designer that can tell us which approach is better? 
I don't mind any, I just think is it good to specify exactly what icon we refer to in the legend.
![image](https://user-images.githubusercontent.com/59687465/98090113-e830e680-1e83-11eb-8df1-ecb9067987fa.png)

Reply:
Hi @natocesarrego,

Please let me know what the solution would be, and if I can send to our customer this solution. 

Regards.

Reply:
Sorry, @georgel-pop-lr, I believe that I wasn't 100% clear on my previous comment, therefore let me provide to you more details about my thoughts. 🙂 

My suggestion was change the solution proposed by you to something like the following:
```
// Language.properties

fields-required=Fields with * are required

// modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/index.js

<p aria-hidden="true" className="text-secondary">
    {Liferay.Language.get('fields-required')}
</p>
```
This same solution is already used in portal, as you can see [here](https://github.com/liferay/liferay-portal/blob/6e486ffaee3faa9e0c7495781ebeab5518d6a94b/modules/apps/blogs/blogs-api/src/main/resources/content/Language.properties#L2), for example.

Is it clear to you now the goal of my previous comment?

Additionally, I understand that the actual solution can be delivered to our customer as a hotfix. After all, we're very close to finish the review of this pull. 😄

Reply:
Hi @natocesarrego,

You have been very clear, but I don't think we are referring to the same case here. 
The case explained by you in the last comment is referring to file extensions.
In our case we add a legend referring to an icon description, if we add * might make some sense, but is not completely clear we refer to that icon and this is what our client asks.

Regards.

Reply:
The solution is good.

I only see that the left alignment of all elements is wrong. But that is another issue.

Reply:
Hi @natocesarrego,

Why can't we go with the first solution proposed by Lexicon team:
![image](https://user-images.githubusercontent.com/59687465/98256785-4393cf00-1f7f-11eb-87fc-d1ee6abcfec4.png)
and like this we can have one translation key, and we keep the icon in the beginning?

Regards.

Reply:
The solution proposed by Lexicon team looks good to me, but Forms team designer proposed the message `Fields with * are required`. I'll explain to him the current situation and I'll ask if we can move on with Lexicon team's proposal. I get back to you soon. 🙂

Reply:
Hey @georgel-pop-lr, Forms team designer is on PTO until the middle of next week, even though I don't think we should wait so long to take a simple decision like this, thereby go ahead and use the Lexicon team's suggestion.

Thanks.

Reply:
Hi @natocesarrego,

You refer to the first option proposed by Lexicon
 https://user-images.githubusercontent.com/59687465/98256785-4393cf00-1f7f-11eb-87fc-d1ee6abcfec4.png

Regards.

Reply:
Yes!